### PR TITLE
[bug]fix the monitor status is abnormal

### PR DESCRIPTION
**Pull request by Yanshuming1**
fix the monitor status is abnormal

### CODE REVIEW

**Comment by tomsun28** [comment]
fix https://github.com/apache/hertzbeat/issues/2062

**Comment by tomsun28** [comment]
hi, this relates the common alarm part. Please wait that we need time to conduct a detailed test, review.

File: alerter/src/main/java/org/apache/hertzbeat/alert/reduce/AlarmConvergeReduce.java [hunk 1]
```diff
@@ -55,25 +55,33 @@ public AlarmConvergeReduce(AlertConvergeDao alertConvergeDao) {
     @SuppressWarnings("unchecked")
     public boolean filterConverge(Alert currentAlert) {
         // ignore monitor status auto recover notice
-        if (currentAlert.getTags() != null && currentAlert.getTags().containsKey(CommonConstants.IGNORE)) {
-            return true;
-        }
-        if (currentAlert.getStatus() == CommonConstants.ALERT_STATUS_CODE_RESTORED) {
+        if ((currentAlert.getTags() != null && currentAlert.getTags().containsKey(CommonConstants.IGNORE))
+                || currentAlert.getStatus() == CommonConstants.ALERT_STATUS_CODE_RESTORED) {
```
Comment (tomsun28) [inline alerter/src/main/java/org/apache/hertzbeat/alert/reduce/AlarmConvergeReduce.java:59]:
```suggestion
        if (currentAlert.getStatus() == CommonConstants.ALERT_STATUS_CODE_RESTORED) {
```
status always is the ALERT_STATUS_CODE_RESTORED when tags has CommonConstants.IGNORE

**Review by tomsun28 — APPROVED** [review]
👍 lgtm!

File: alerter/src/main/java/org/apache/hertzbeat/alert/reduce/AlarmConvergeReduce.java (hunk 1 above)
Comment (Yanshuming1) [reply alerter/src/main/java/org/apache/hertzbeat/alert/reduce/AlarmConvergeReduce.java:59]: In both cases, the data in the map needs to be deleted，if remove （|| currentAlert . getStatus () == CommonConstants . ALERT_STATUS_CODE_RESTORED ），So when the label is not IGNORE，Unable to clear cache

**Pull request merged**